career-mentorship-program/2021-spring/matching_code.py

```diff
@@ -175,6 +194,21 @@ class Mentee(Student):
     # Which areas of the recruitment process the mentee wants help with
     desired_topics: Set[str]
 
+    # Hours per week the mentor is able to dedicate
+    available_time: int
+
+    # What are their hobbies/interests
```

Comment:
We can leave out the comments on the fields when they are rather self-explanatory, such as for `hobbies`, `gender`, and `race`.